cpp/src/slice2cs/Gen.cpp

```diff
@@ -1110,23 +1199,27 @@ Slice::Gen::TypesVisitor::visitDictionary(const DictionaryPtr& p)
     string name = "global::System.Collections.Generic." + genericType + "<" + keyS + ", " + valueS + ">";
 
     _out << sp;
+    ostringstream summary;
+    summary << "Provides methods to marshal and unmarshal a <c>" << p->name() << "</c>.";
+    writeHelperDocComment(p, summary.str(), "helper class");
```

Comment:
Should this be "dictionary helper class", like we do for Proxy? otherwise can we just always use "helper class"?

Currently is "helper class" for all but proxy.

Reply:
Yes, we should qualify it: sequence helper class, dictionary helper class etc.